packages/zosfiles/src/methods/search/doc/ISearchOptions.ts

```diff
@@ -45,4 +45,7 @@ export interface ISearchOptions {
     /* A function that, if provided, is called with a list of data sets and members that are about to be searched. */
     /* If true, continue search. If false, terminate search. */
     continueSearch?: (dataSets: IDataSet[]) => Promise<boolean> | boolean;
+
+    /* A function that gets called to validate whether or not to abort if a timeout isn't specified */
+    abortSearch?: () => boolean;
```

Comment:
This could also be true for other extending applications, since searching hundreds or thousands of data sets or members could take a very long time, and there is no way for a calling application to stop the inner code running in an asynchronous function once it starts, unless that function provides a method to terminate it early.

Reply:
Thanks for the clarification.